### PR TITLE
forgot ograph on other pages

### PR DESCRIPTION
**Pull request by shaqsingh**
should sleep more

### CODE REVIEW

**Comment by dcwalk** [comment]
I actually think this is a great candidate for taking advantage of a jekyll template? We could auto-populate from YAML frontmatter as well? Mind if I mock something up?

I don't want to be a block tho, so I can revisit in another PR

**Comment by shaqsingh** [comment]
@dcwalk I will def revisit after exams are done! I don't mind if you mock something up though

File: index.html [hunk 1]
```diff
@@ -12,12 +12,22 @@
...
     <link rel="canonical" href="{{ site.production_url }}{{ page.url | replace:'index.html',''}}">
     <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="/feed.xml">
-	  <link rel="stylesheet" href="/assets/tachyons.min.css"/>
-      <link href="https://fonts.googleapis.com/css?family=Open+Sans:600,700|Open+Sans+Condensed:300,700|Source+Sans+Pro:300,400,400i,600" rel="stylesheet">
-	</head>
+	<link rel="stylesheet" href="/assets/tachyons.min.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:600,700|Open+Sans+Condensed:300,700|Source+Sans+Pro:300,400,400i,600" rel="stylesheet">
+    <meta property="og:title" content="The First 100 Days And Counting"/>
+    <meta property="og:type" content="article"/>
+    <meta property="og:url" content="https://100days.envirodatagov.org/index.html"/>
```
Comment (patcon) [inline index.html]: A couple lines above, I used this pattern to string index.html from a url (which is also intended to be canonical), so we could use the same thing here:
```
{{ site.production_url }}{{ page.url | replace:'index.html',''}}
```

File: epa-under-siege-peer-reviews.html [hunk 2]
```diff
@@ -9,6 +9,17 @@
         <link href="https://fonts.googleapis.com/css?family=Open+Sans:600,700|Open+Sans+Condensed:300,700|Source+Sans+Pro:300,400,400i,600" rel="stylesheet">
         <link rel="shortcut icon" href="/assets/favicon.ico" type="image/x-icon">
         <link rel="icon" href="/assets/favicon.ico" type="image/x-icon">
+        <meta property="og:title" content="Part 1 - Peer Reviews"/>
+        <meta property="og:type" content="article"/>
+        <meta property="og:url" content="https://100days.envirodatagov.org/epa-under-siege-peer-reviews.html"/>
+        <meta property="og:image" content="assets/ch-1cover.jpg"/>
+        <meta property="og:description" content="The EPA Under Siege: Trump’s Assault in History and Testimony"/>
+        <meta name="twitter:site" value="@EnviroDGI">
+        <meta property="twitter:title" content="Part 1 - Peer Reviews"/>
```
Comment (patcon) [inline epa-under-siege-peer-reviews.html]: I believe Twitter cards use "name" not "property":
https://dev.twitter.com/cards/getting-started

> Open Graph protocol also specifies the use of `property` and `content` attributes for markup (`<meta property="og:image" content="http://example.com/ogp.jpg"/>`) while Twitter cards use `name` and `content`. 

This is mentioned next:

> Twitter’s parser will fall back to using property and content, so there is no need to modify existing Open Graph protocol markup if it already exists.

But I believe, as demonstrated in the example below it on the source page, that's just in reference to that fact that `og:*` properties can use used instead, and twitter will parse those for certain properties, as indicated on the markup tag reference page (the "OpenGraph" column): https://dev.twitter.com/cards/markup

**Comment by patcon** [comment]
Hey @shaqsingh, a couple blockers, but I know you're in exams, so I'm happy to make the changes myself. Since I was the hold-up, I'll still merge asap so that we get a deploy out, and we're not holding up the team :)

File: epa-under-siege-peer-reviews.html [hunk 3]
```diff
@@ -9,6 +11,15 @@
         <link href="https://fonts.googleapis.com/css?family=Open+Sans:600,700|Open+Sans+Condensed:300,700|Source+Sans+Pro:300,400,400i,600" rel="stylesheet">
         <link rel="shortcut icon" href="/assets/favicon.ico" type="image/x-icon">
         <link rel="icon" href="/assets/favicon.ico" type="image/x-icon">
+        <meta property="og:title" content="Part 1 - Peer Reviews"/>
+        <meta property="og:type" content="article"/>
+        <meta property="og:url" content="{{ site.production_url }}{{ page.url | replace:'index.html',''}}"/>
+        <meta property="og:image" content="assets/ch-1cover.jpg"/>
```
Comment (patcon) [inline epa-under-siege-peer-reviews.html]: Typo. Should be `assets/ch1-cover.jpg`

**Comment by patcon** [comment]
Pull the "heroku review app"-related updates from master, so that we can use those. Since we have a public page, I'm now using the [Twitter Card Validator](https://cards-dev.twitter.com/validator) to find some other things to tweak.

**Comment by patcon** [comment]
Tried to test for facebook too, but it's debugger follows the redirect to our `og:url`, which we set to the canonical url. This confuses the debugger, as the canical url doesn't have OpenGraph metadata on it yet, but should be good once deployed

https://developers.facebook.com/tools/debug/sharing

File: epa-under-siege-peer-reviews.html [hunk 4]
```diff
@@ -43,7 +54,7 @@ <h2 class="f3 fw3 ttu white-80">The EPA Under Siege: Trump’s Assault in Histor
             <div id="contr" class="vh-25 w-100 dt tc tile pv0-l pv2-m pv4-ns">
                 <div class="tile-content">
                     <h3 class="f3 ttu white">Contributors</h3>
-                    <p class="f4 fw2 white lh-normal">Christopher Sellers, Lindsey Dillon, Jennifer Liss Ohayon, Nick Shapiro, Marianne Sullivan, Chris Amoss, Stephen Bocking, Phil Brown, Vanessa De la Rosa, Jill Harrison, Sara Johns, Katherine Kulik, Rebecca Lave, Michelle Murphy, Liza Piper, Lauren Richter, Sara Wylie</p>
+                    <p class="f4 fw2 white lh-normal">Dr. Richard Andrews, Dr. Nancy Langston, Dr. Jay Turner</p>
```
Comment (patcon) [inline epa-under-siege-peer-reviews.html:57]: Can anyone tell me if this reduction in names was intentional? It's happening in a commit whose message doesn't reflect this being the intention

Comment (patcon) [reply epa-under-siege-peer-reviews.html:57]: @shaqsingh confirmed that it's intentional, so nevermind!

**Comment by patcon** [comment]
Deployed.

Confirmed on live site that Twitter validator and Facebook debugger work fine on the main page:
https://cards-dev.twitter.com/validator
https://developers.facebook.com/tools/debug/sharing/?q=https%3A%2F%2F100days.envirodatagov.org%2F

**Pull request merged**